bug_update_page.php

```diff
@@ -203,7 +203,7 @@
 ?>
 			<tbody>
 <?php
-event_signal( 'EVENT_UPDATE_BUG_FORM_TOP', array( $t_bug_id, true ) );
+event_signal( 'EVENT_UPDATE_BUG_FORM_TOP', array( $t_bug_id ) );
```

Comment:
There was the basic/advanced page diferentiation. In #0010890 (2009), it was refactored and a lot of things got modified. These must have slipped out of the clean-up.
Most of the page related events had this parameter indicating if it was signaled from a basic or advanced type page.